plugins/tpm/fu-plugin-tpm.c

```diff
@@ -41,23 +41,22 @@ fu_plugin_add_security_attrs (FuPlugin *plugin, FuSecurityAttrs *attrs)
 	g_autoptr(FwupdSecurityAttr) attr = NULL;
 
 	/* create attr */
-	attr = fwupd_security_attr_new ("org.trustedcomputinggroup.Tpm");
+	attr = fwupd_security_attr_new (FWUPD_SECURITY_ATTR_ID_TPM_VERSION_20);
 	fwupd_security_attr_set_plugin (attr, fu_plugin_get_name (plugin));
 	fwupd_security_attr_set_level (attr, FWUPD_SECURITY_ATTR_LEVEL_CRITICAL);
-	fwupd_security_attr_set_name (attr, "TPM");
 	fu_security_attrs_append (attrs, attr);
 
 	/* check exists, and in v2.0 mode */
 	if (!data->has_tpm) {
-		fwupd_security_attr_set_result (attr, "Not found");
+		fwupd_security_attr_set_result (attr, FWUPD_SECURITY_ATTR_RESULT_NOT_FOUND);
 		return;
 	}
 	if (!data->has_tpm_v20) {
-		fwupd_security_attr_set_result (attr, "Not in v2.0 mode");
+		fwupd_security_attr_set_result (attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENABLED);
 		return;
 	}
 
 	/* success */
 	fwupd_security_attr_add_flag (attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS);
-	fwupd_security_attr_set_result (attr, "v2.0");
+	fwupd_security_attr_set_result (attr, FWUPD_SECURITY_ATTR_RESULT_ENABLED);
```

Comment:
I think this should actually be found, looking again.  We have no guarantees that anything in the OS is using it, just that the platform has it.

Reply:
Fixed. Feel free to fix up any of these as required post merge.